config/config/sc-config.yaml

```diff
@@ -177,7 +177,7 @@ elasticsearch:
     # Where to find roles
     roles_key: groups
     # Scope
-    scope: "openid profile email"
+    scope: "openid profile email groups"
```

Comment:
I'm not sure that this should be the default config as I believe this scope does not work when using the google oidc connector. In the [dev flavor](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/flavors/dev-sc.yaml#L15) this won't get overridden, whereas it is overridden in the [prod flavor](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/flavors/prod-sc.yaml#L26).

Reply:
good point - will change it back to what it was and add in a comment for some context around groups